### ISSUE TITLE
Weather warnining with ED2 at US-WCr using web interface

### ISSUE THREAD

**Issue by serbinsh** [PecanProject/pecan#526]
WARNING : Selected dates are not within the bounds of the weather data file you selected.

I get this warning when running ED2 for US-WCr if the end year contains 2006/12/31

But as far as I know the met covers this range?

**Comment by serbinsh** [comment]
@robkooper Where do we set checks, like this, for met data?

**Comment by robkooper** [comment]
This is done at: https://github.com/PecanProject/pecan/blob/master/web/04-runpecan.php#L77

**Comment by mdietze** [comment]
@serbinsh, take a look at the actual data on cdiac and let us know. Ryan and I were looking at an example like that this morning (Sylvania) and it turned out that the server had the csv files but not the nc files. Right now the workflow is set up for the nc files. It wouldn't be hard to add a BrownDog csv to nc conversion in download.ameriflux, but it might be simpler to just let ameriflux know they screwed up their processing.

**Comment by serbinsh** [comment]
@mdietze Just to be clear, I was assuming that the web interface workflow was basing the met data and ranges on the inputs table, which I populated with the add.data script.  There I am telly BETY where the ED_MET_HEADER is as well as the ranges of the met files.  I assumed that was what it was using for the runs.  In my met data, I have data from 1998-2006, for example, for US-WCr; this comes from the NACP met drivers.

Does the web version of PEcAn query the CDIAC to determine date ranges for available data? I feel like I am missing a step.

Pardon my ignorance!

**Comment by mdietze** [comment]
@serbinsh no, I misunderstood. I thought you were getting an error in the met.process workflow in building the met from Ameriflux on the fly. When it does that it goes to http://cdiac.ornl.gov/ftp/ameriflux/data/Level2/Sites_ByID/ to get the data but it doesn't do any checks on date range or availability, it just returns an error if the data's not there. 

It appears that Rob's code does date checks with < and > not <= and >= so if the end date is identical to the met date it would fail. Seems like a simple fix.

**Comment by serbinsh** [comment]
@mdietze Agreed.  I tested and if I set the end date to, say, 30 instead of 31 it works fine.  So a simple change to <= and >= will correct.  I will look into it

**Comment by serbinsh** [comment]
@tonygardella am I to understand this was fixed, yes?

**Comment by tonygardella** [comment]
@serbinsh Not sure. I can check it out later.
